include/seqan3/io/views/detail/take_exactly_view.hpp

```diff
@@ -255,7 +255,7 @@ class view_take_exactly<urng_t, or_throw>::basic_iterator :
     ~basic_iterator() = default;                                      //!< Defaulted.
 
     //!\brief Constructor that delegates to the CRTP layer.
-    constexpr basic_iterator(base_base_t const & it) noexcept(noexcept(base_t{it})) : base_t{std::move(it)}
+    constexpr basic_iterator(base_base_t it) noexcept(noexcept(base_t{it})) : base_t{std::move(it)}
```

Comment:
Why do you remove the `const &` instead of just removing the `std::move` call?

Reply:
Both would work, iterators are usually (should be) cheap to copy.
It's more in line with the next constructor just below this one, where we take the `base_base_it` by value and then `std::move` it.

Reply:
Ok, thanks